src/nodes/variable.ts

```diff
@@ -113,45 +113,43 @@ export class CVariableAllocation extends CTemplateBase {
 }
 
 @CodeTemplate(`
-{#statements}
```

Comment:
I think `{#statements}` was there for a reason...

Reply:
I don't really know, but i know the tests pass... I removed statements because it moved it above the actual return code.

Reply:
I tested it and basically the main side effect is that it inserts empty lines in random places if destructors are not within `{#statements}` block. so in the best case scenario we probably want to keep `{#statements}`.

I think if `{returnTypeAndVar} = {expression};` line is also put into `{#statements}` then it will appear above destructors.

Reply:
> I think if {returnTypeAndVar} = {expression}; line is also put into {#statements} then it will appear above destructors.

my assumption was incorrect

I guess we will have to get rid of `{#statements}` after all, but it would still be great to fix the whitespaces somehow. I'm looking into it now

Reply:
General whitespace handling improvements were done in c27ad6dec619d0a196abe3c671ceacdefcd95537

However, I noticed there's another side effect of removing `{#statements}` from destructors: it adds destructors after `return` in some cases. It is an unreachable code, that's why tests weren't failing, but it's quite bad still, obviously.

I'll have to investigate and fix it.